src/components/preview_component/preview.sh

```diff
@@ -99,9 +99,13 @@ run_preview() {
     # if running end to end script with sbatch then use
     # sbatch to take advantage of multiple cores 
     if "$UseSlurm"; then
-        # set number of cores to run preview with
+        # set number of cores and memory to run preview with
         if "$isAWS"; then
-            sed -i -e "s:#SBATCH -c 8:#SBATCH -c ${cpu_count}:g" ${InversionPath}/src/inversion_scripts/imi_preview.py
+            sed -i -e "s:#SBATCH -c 8:#SBATCH -c ${cpu_count}:g" \
```

Comment:
I don't see `#SBATCH -c 8` in the header of `imi_preview.py`. There are statements for `-n` and `-N` but not `-c`. Do we need `sed` lines to comment those out, like the line below for `--mem`?

Also, do we need sed to add in the `#SBATCH -c ${cpu_count}` line? Or can it use the `-n` flag instead, like this:

`sed -i -e "s:#SBATCH -n 1:#SBATCH -n ${cpu_count}:g" `